torch/onnx/symbolic_opset9.py

```diff
@@ -5009,11 +5009,11 @@ def tolist(g, input, dim_val, elem_ty_val):
     def device(ctx: torch.onnx.SymbolicContext, g, *inputs, **kwargs):
         n = ctx.cur_node
 
-        if n.output().type().kind() == "_C.DeviceObjType":
+        if n.output().type().kind() == "DeviceObjType":
```

Comment:
We should really test against the actual type objects, not the strings using:
`isinstance(n.output().type(), DeviceObjType)`

Then typos would be caught much more easily.

Reply:
Thanks for the suggestion! Drafted #78114